alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/resthandler/RestIndexDestinationAction.kt

```diff
@@ -145,6 +155,18 @@ class RestIndexDestinationAction(
             }
         }
 
+        private fun onUpdateMappingsResponse(response: AcknowledgedResponse) {
+            if (response.isAcknowledged) {
+                log.info("Updated  ${ScheduledJob.SCHEDULED_JOBS_INDEX} with mappings.")
+                scheduledJobIndexUpdated = true
+                prepareDestinationIndexing()
+            } else {
+                log.error("Update ${ScheduledJob.SCHEDULED_JOBS_INDEX} mappings call not acknowledged.")
+                channel.sendResponse(BytesRestResponse(RestStatus.INTERNAL_SERVER_ERROR,
+                        response.toXContent(channel.newErrorBuilder(), ToXContent.EMPTY_PARAMS)))
```

Comment:
This will give a blank response back to the user. We should construct a basic error message with the `channel.newErrorBuilder()`.

Reply:
Will change it.